emails/views.py

```diff
@@ -604,6 +609,12 @@ def _handle_bounce(message_json):
     bounce = message_json.get('bounce')
     bounced_recipients = bounce.get('bouncedRecipients')
     for recipient in bounced_recipients:
+        recipient_address = recipient.pop('emailAddress', None)
+        recipient_address = parseaddr(recipient_address)[1]
```

Comment:
```suggestion
        recipient_full_address = recipient.pop('emailAddress', None)
        recipient_address = parseaddr(recipient_full_address)[1]
```
`emailAddress` value is looked up later in the code so we should assign this as a separate variable to use it in the try/catch block.